docs/basics/basic_phases.md

```diff
@@ -0,0 +1,168 @@
+# Phases
+
+These are different types of phases based on the mode of starting new users.  For more detail, please see [phases]({{ "/docs/concepts.html#phases" | absolute_url }})
```

Comment:
Btw., have you considered presenting the simplified form with `constantPerSec` properties inlined as top-level ones?